docs/docs/index.md

```diff
@@ -57,7 +57,7 @@ var myChart = new Chart(ctx, {
 
 ## Contributing
 
-Before submitting an issue or a pull request to the project, please take a moment to look over the [contributing guidelines](./developers/contributing.md) first.
+Before submitting an issue or a pull request to the project, please take a moment to look over the [contributing guidelines](./docs/developers/contributing.md) first.
```

Comment:
I tested locally and this change isn't needed. The previous link worked locally and on https://www.chartjs.org/docs/master/#contributing

Reply:
on the live website, it doesn't seem to work

Reply:
When I clicked on the link to the contributing guidelines, it went to https://www.chartjs.org/docs/master/developers/contributing

Reply:
On my end here, if you visit https://www.chartjs.org/docs/latest/

and click on the Contributing guidelines link, it goes to https://github.com/chartjs/Chart.js/blob/master/docs/developers/contributing.md which happens to be 404

Reply:
Ah, ok. so `latest` is the last full release, 2.9.3. We're working on v3.0.0 now and we rewrote the doc system during the development. The docs I linked are the ones that the code currently builds where this change isn't needed (the other links were broken in the dev version too).

Reply:
it's okay then, can you make the adjustment that fits the v3.0.0, thanks

Reply:
Done. I merged on the command line and did the change there. Thanks for the contribution 😄

Reply:
you are welcome sir